# Maxwells-duiker-hepatitis-B-virus
# A novel orthohepadnavirus identified in a dead Maxwell’s duiker (Philantomba maxwellii) in Taï National Park, Côte d’Ivoire

# Uploaded here is an effort to share the results of our phylogenetic analysis seeking to understand the relationship of our new 
# virus with other known hepadnaviruses 

# To assess the phylogenetic position of the Maxwell’s duiker orthohepadnaviruses within hepadnaviruses, we downloaded representative hepadnaviral genomes from GenBank and divergent viruses fish viruses described by Lauber, et al. [22]. We followed Aghazadeh, Shi, Barrs, McLuckie, Lindsay, Jameson, Hampson, Holmes and Beatty [13] and extracted the Polymerase open reading frame (ORF) and translated sequences to amino acids and aligned sequences using MAFFT (v7.307). The core and surface protein ORFs were more divergent in these genomes and so we targeted only the orthohepadnaviruses, using the hepadnavirus from the bluegill (GenBank accession NC_030445) as an outgroup [following: 13]. For each amino acid alignment, we selected conserved blocks using Gblocks [23], as implemented in SeaView V4 [24]. To construct a phylogeny, we used PhyML with smart model selection [25], selecting the best model of amino evolution using the Bayesian Information Criterion and subtree pruning and regrafting (SPR) as the tree improvement approach, and otherwise default settings (Selected model	: polymerase ORF=LG+G+I+F; core ORF=JTT+G; surface ORF=JTT+G+F). We also reran the analysis using an alignment comprising only orthohepadnaviruses and excluding the highly divergent outgroup, to check whether including this outgroup affected tree structure and assess potential long-branch attraction effects (Selected model: polymerase ORF=LG+G+I+F; core ORF: JTT+G; surface ORF: AB+G+I+F). We estimated the best-fitting root of these phylogenies using the heuristic residual mean squared function in the program TempEst, which minimizes the variance of root-to-tip distances [26]. To further assess the confidence in our phylogenetic trees, BMCMC analyses were run on each amino acid alignment using BEAST v1.10.4 under the assumption of a relaxed lognormal molecular clock and with tree shape modeled according to a birth-death speciation model and the amino acids substitution model supported by PhyML’s smart model selection (XML and tree files are available on github) [27]. We examined the output of three runs for convergence and appropriate sampling of the posterior using Tracer v1.7.1 [28] before merging runs using LogCombiner v1.10.4 [29]. The best representative tree was then identified from the posterior set of trees and annotated with TreeAnnotator v1.10.4 (distributed with BEAST). 

